micrometer-registry-new-relic-telemetry/build.gradle

```diff
@@ -2,6 +2,11 @@ plugins {
     id 'io.micronaut.build.internal.micrometer-registry'
 }
 
+repositories {
```

Comment:
can we move the repositories block to `micrometer-registry` gradle file?

Reply:
I reverted the subproject build changes and moved the repositories block to `micrometer-module.build` (rather than `micrometer-registry.build` which won't entirely work).